conans/client/conan_api.py

```diff
@@ -953,6 +953,29 @@ def get_remote_by_name(self, remote_name):
 Conan = ConanAPIV1
 
 
+def install_input(profile_name, settings, options, env, cwd, install_folder, client_cache, output):
+    try:
+        graph_info = GraphInfo.load(install_folder)
+        graph_info.profile.process_settings(client_cache, preprocess=False)
+    except Exception:
+        if install_folder:
+            raise ConanException("Failed to load graphinfo file in install-folder: %s"
+                                 % install_folder)
+        graph_info = None
+
+    if profile_name or settings or options or profile_name or env or not graph_info:
+        if graph_info:
+            # FIXME: Convert to Exception in Conan 2.0
+            output.warn("Settings, options, env or profile specified. "
+                        "GraphInfo found from previous install won't be used: %s"
+                        % install_folder)
```

Comment:
Add a recomendation to the user. What to do? Do not provide the settings, options, env, profile?

Reply:
Ok

Reply:
Please, add the comment about this, what should the user do? Delete that `GraphInfo` thing (is it a file?)? is it better not to provide settings, options, env,...? Or maybe it is ok and the two sources will be merged...